src/main/java/com/gudimov/movieland/controller/MovieController.java

```diff
@@ -59,5 +60,20 @@
         return new ResponseEntity<>(movieJson, HttpStatus.OK);
 
     }
+
+    @RequestMapping(value = "/genre/{genreId}")
+    @ResponseBody
+    public ResponseEntity<String> getMovieByGenreId(@PathVariable int genreId){
+        log.info("Sending request to get movie by genre id = {}", genreId);
+        List<Movie> movies = movieService.getByGenreId( genreId);
+        List<MovieDto> movieDtos = new ArrayList<>();

```

Comment:
better way is to have separate abstract DTO converter class with static methods to convert Object and List. Conversion is a separate job, not for controller.